WebDriverAgentLib/FBAlert.m

```diff
@@ -191,7 +191,7 @@ - (BOOL)acceptWithError:(NSError **)error
     ? [[[FBErrorBuilder builder]
         withDescriptionFormat:@"Failed to find accept button for alert: %@", self.alertElement]
      buildError:error]
-    : [acceptButton fb_tapWithError:error];
+    : [acceptButton fb_tapCoordinateWithError:error];
```

Comment:
Any reason this should be changed? Maybe we could simply do https://github.com/appium/WebDriverAgent/pull/540/files#diff-f03b155e56b1e1fbb9b99d5aecdd8a9270b2a422335987bfab3c7f104344fb27R32 for the corresponding test suites or to the base test suite.

Reply:
yea, i noticed it i needed to set in FBIntegrationTestCase.m